actions/promote/createBatch.js

```diff
@@ -111,7 +111,11 @@ async function main(params) {
 async function createBatch(batchManager, appConf) {
     const { sp } = await getConfig();
     const options = await getAuthorizedRequestOption({ method: 'GET' });
-    const promoteIgnoreList = appConf.getPromoteIgnorePaths();
+    let promoteIgnoreList = appConf.getPromoteIgnorePaths() || [];
```

Comment:
The defaults are already added here:
https://github.com/adobecom/milo-fg/blob/main/actions/appConfig.js#L159
Also we should not default this to empty array.